### PR TITLE
What's New and Upgrade Steps for 6.0

### PR DESCRIPTION
**Pull request by bhariharan**

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -34,25 +34,47 @@ If you have problems building any of the projects, take a look at the online [FA
...
+
+**SmartSync Enhancements**
+- Beginning in Mobile SDK 6.0, you can define sync configuration files and assign names to sync configurations.
+- You can use sync names to run, edit, or delete a saved sync operation.
+- You can define “sync down” and “sync up” operations through configuration files rather than code.
+- To define sync operations for the default global store, provide a file named `globalsyncs.json`.
+- To define sync operations for the default user store, provide a file named `usersyncs.json`.
+
+**Other Enhancements**
+- Mobile SDK 6.0 adds support for a developer settings screen in DEBUG builds.
```
Comment (wmathurin) [inline readme.md]: New in Mobile SDK 6.0, we are adding dev support screens. You do not need to write any code to use them. To access the dev support dialog, shake your device or select "Shake Gesture" inside the Hardware menu in the iOS Simulator. You can also use the ^⌘z keyboard shortcut when your app is running in the iOS Simulator.

File: UPGRADE.md [hunk 2]
```diff
@@ -1,13 +1,7 @@
-## Upgrade steps from v. 5.2.x to v. 5.3 for native/hybrid/react native apps
+## Upgrade steps from v. 5.3.x to v. 6.0 for native/hybrid/react native apps
 
 To upgrade native, hybrid and react native apps, we strongly recommend creating a new app from the app templates in [the forceios npm package](https://npmjs.org/package/forceios), then migrating the artifacts specific to your app into the new template. The other recommended approach would be to upgrade using [Cocoapods](https://github.com/forcedotcom/SalesforceMobileSDK-iOS-Specs).
```
Comment (wmathurin) [inline UPGRADE.md]: Fro hybrid apps, they will need to use forcehybrid. And for react native apps, they should use forcereact.

**Comment by brandonpage** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Error">
          1 Error
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:no_entry_sign:</td>
      <td data-sticky="false">Tests have failed, see below for more information.</td>
    </tr>
  </tbody>
</table>
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Warning">
          1 Warning
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">Please provide a summary in the Pull Request description</td>
    </tr>
  </tbody>
</table>

### Tests: 

Classname | Name|
--- | ---|
SmartSyncTestSuite | testSyncSObjectDetectConflictDelete|
SmartSyncTestSuite | testSyncSObjectDetectConflictUpdate|
SmartSyncTestSuite | testSyncSObjectRetrieve|
SyncManagerTests | testCleanResyncGhostsForMRUTarget|
SyncManagerTests | testCleanResyncGhostsForMRUTarget|
SyncManagerTests | testCleanResyncGhostsForMRUTarget|


## Current coverage for SalesforceAnalytics.framework is `67.16%`
✅ *No files affecting coverage found*

---
## Current coverage for SalesforceHybridSDK.framework is `55.12%`
✅ *No files affecting coverage found*

---
## Current coverage for SalesforceSDKCore.framework is `50.86%`
✅ *No files affecting coverage found*

---
## Current coverage for SmartStore.framework is `64.15%`
✅ *No files affecting coverage found*

---
## Current coverage for SmartSync.framework is `79.73%`
✅ *No files affecting coverage found*

---

> Powered by [xcov](https://github.com/nakiostudio/xcov)

<p align="right" data-meta="generated_by_CI-result">
  Generated by :no_entry_sign: <a href="http://danger.systems/">Danger</a>
</p>

**Comment by bhariharan** [comment]
@wmathurin Done.

**Comment by bhariharan** [comment]
Got it reviewed by Richard offline.

**Pull request merged**